frontend/packages/pipelines-plugin/locales/ja/pipelines-plugin.json

```diff
@@ -28,61 +30,61 @@
   "An error occurred while retrieving the requested logs.": "要求されたログの取得中にエラーが発生しました。",
   "Unknown error retrieving logs": "ログの取得に関する不明なエラー",
   "Download": "ダウンロード",
-  "Download All": "すべてをダウンロード",
+  "Download all": "すべてをダウンロード",
   "Collapse": "縮小",
   "Expand": "展開",
   "Failure - check logs for details.": "失敗しました。ログで詳細を確認してください。",
   "Message": "メッセージ",
-  "Log Snippet": "ログスニペット",
+  "Log snippet": "ログスニペット",
   "Task Runs": "タスク実行",
```

Comment:
Should this be sentence case? `Task runs`

Reply:
It seems like this a defined model name.